Client/SpeechHandler.py

```diff
@@ -34,7 +35,14 @@
 HOST = input("Multicraft Server IP: ").split(':')[0] # use socket.gethostbyname(socket.gethostname()) if you are also hosting
 PORT = 5001
 CLIENT_SOCKET = socket.socket()
-CLIENT_NAME = input("Your Full Minecraft Java UUID: ")
+mc_username = input("Your Full Minecraft Username: ")
+with urllib.request.urlopen(f"https://api.mojang.com/users/profiles/minecraft/{mc_username}") as response:
+    mc_profile = response.read().decode("utf-8")
+try:
+    CLIENT_NAME = json.loads(mc_profile)["id"]
+    print(f"Connecting as {CLIENT_NAME}")
+except ValueError:
```

Comment:
In the case that api.mojang.com/users/profiles/minecraft/{mc_username} responds with a 204 (No Content), the type of error thrown by line 42 is not a ValueError. I'd change line 44 to account for any exception, but the exception I ran into when testing was a json.decoder.JSONDecodeError.

Reply:
Also upon exception (line 45, when "Invalid Username" is printed), it may be better to exit the program (using exit(1)) as the server would not be able to identify the user in this situation.

Reply:
I think JSONDecodeError is a type of ValueError but I changed it to handle all errors and to exit.